scanners/semgrep/Chart.yaml

```diff
@@ -18,7 +18,7 @@ version: "v3.1.0-alpha1"
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.71.0"
+appVersion: "v0.72.0"
```

Comment:
@Ilyesbdlala why is the bot adding the "v"? The correct image tag is without the "v", so this makes the tests fail.